packages/runtime/container-runtime/src/summaryGenerator.ts

```diff
@@ -236,6 +236,13 @@ export class SummaryGenerator {
         let summaryData: SubmitSummaryResult | undefined;
         let generateTelemetryProps: Record<string, string | number | boolean | undefined> = {};
         try {
+            const generateSummaryEvent = PerformanceEvent.start(logger, {
+                eventName: "Summarize",
+                fullTree,
+                timeSinceLastAttempt: Date.now() - this.heuristicData.lastAttempt.summaryTime,
```

Comment:
Don't regenerate these times again. Store it in a variable above (before summarizeEvent is declared) and use it for both the events.